src/http/server.ts

```diff
@@ -331,10 +332,21 @@ export class Server {
         if (isNaN(e.code)) {
           e.code = 500;
         }
-        return new Response(e.stack, {
-          status: e.code,
-          headers: response.headers,
-        });
+
+        if (exception) {
+          try {
+            await (new exception()).catch(e, originalRequest, response);
```

Comment:
feel like the `catch` method should be static?

Reply:
> Also in regards to [#592 (comment)](https://github.com/drashland/drash/pull/592#discussion_r777264575), i think eric means, instantiate the exception class in the server constructor:
> 
> ```ts
> #exception: CustomException|null = null
> constructor(...) {
>   if (options.exception) {
>     this.#exception = new options.exception()
>   }
> }
> ...
> 
> const exception = this.#exception
> return function () {
>   exception.catch(...)
> }
> ```

Yup, this is what he means. Essentially the https://github.com/drashland/drash/pull/592/commits/8d4c75ea41671200f77735dc62f9feaeabf756fd was wrongly done.

> But that wouldn't be a problem if the catch method was static anyways

I would go against this. The purpose is that the users can extends the behaviour without breaking it. Static means the previous behaviour is lost.

Reply:
Resolving because the scalability of not making it static is better